vector/src/main/res/layout/item_bottom_sheet_action.xml

```diff
@@ -82,4 +82,27 @@
         tools:ignore="MissingPrefix"
         tools:visibility="visible" />
 
+
+    <TextView
+        android:id="@+id/actionBetaTextView"
+        style="@style/Widget.Vector.TextView.Caption"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:background="@drawable/notification_badge"
+        android:backgroundTint="@color/palette_azure"
+        android:gravity="center"
+        android:paddingStart="10dp"
+        android:paddingTop="3dp"
+        android:paddingEnd="10dp"
+        android:layout_marginEnd="1dp"
+        android:paddingBottom="3dp"
+        android:visibility="gone"
+        tools:visibility="visible"
+        android:text="@string/beta_title_bottom_sheet_action"
+        android:textColor="@color/palette_white"
```

Comment:
to double check, should this use the theme colours? I'm guessing we're not using them because the background colour is hardcoded blue

Reply:
Yeap, we are not using them, its the same for both themes!